src/ApiService/ApiService/Program.cs

```diff
@@ -58,6 +58,54 @@ public class LoggingMiddleware : IFunctionsWorkerMiddleware {
         }
     }
 
+    /// <summary>
+    /// 
+    /// </summary>
+    public class VersionCheckingMiddleware : IFunctionsWorkerMiddleware {
```

Comment:
Do we have unit tests for these changes?

Reply:
Unit tests have been added.